src/test/utils.ts

```diff
@@ -15,4 +15,11 @@ export {
 
 const DEV_SERVER_PORT = 8000;
 
-export const url = (path: string) => `http://localhost:${DEV_SERVER_PORT}/${path}`;
+export const url = (path: string) => `http://localhost:${DEV_SERVER_PORT}/${path.replace(/^\/+/, '')}`;
+
+export const navigate = async(target: puppeteer.Page, path: string) => {
+  await target.goto(url(path));
+  await target.evaluate(async() => {
+    await (window as any).hooks.calm();
```

Comment:
nice! Since we redefine the `window` in `global.d.ts` to optionally be undefined, could it be extended to include the `.hooks`? (or maybe something easier to grep for & less likely to collide like `_RENDERING_HOOKS`)

Reply:
something like what is in commit 72173a85ae90124016470bf9189aed37316133ec